### PR TITLE
planning: open space fix rigid transform bug when extracting obstacles

### PR DESCRIPTION
**Pull request by deidaraho**
1, remove virtual obstacles from open space
2, the rigid transfer to obstacles should be according to original point instead of the obstacle's center.

### CODE REVIEW

File: modules/planning/tasks/deciders/open_space_roi_decider.cc [hunk 1]
```diff
@@ -83,6 +83,11 @@ bool OpenSpaceRoiDecider::VPresentationObstacle() {
 
   if (config_.open_space_roi_decider_config().enable_perception_obstacles()) {
     size_t perception_obstacles_num = obstacles_by_frame_->Items().size();
+    for (const auto &ob : obstacles_by_frame_->Items()) {
+      if (ob->IsVirtual()) {
+        perception_obstacles_num--;
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/open_space_roi_decider.cc]: --perception_obstacles_num;

File: modules/planning/tasks/deciders/open_space_roi_decider.cc [hunk 2]
```diff
@@ -107,13 +112,18 @@ bool OpenSpaceRoiDecider::VPresentationObstacle() {
     const auto &origin_point = frame_->open_space_info().origin_point();
     const auto &origin_heading = frame_->open_space_info().origin_heading();
     for (const auto &obstacle : obstacles_by_frame_->Items()) {
+      if (obstacle->IsVirtual()) { continue; }
```
Comment (xiaoxq) [inline modules/planning/tasks/deciders/open_space_roi_decider.cc]: Expand

File: modules/planning/tasks/deciders/open_space_roi_decider.cc (hunk 1 above)
Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_roi_decider.cc]: > --perception_obstacles_num;

fixed.

File: modules/planning/tasks/deciders/open_space_roi_decider.cc (hunk 2 above)
Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_roi_decider.cc]: > Expand

fixed

**Pull request merged**